doc/api/net.md

```diff
@@ -629,6 +632,22 @@ For [IPC][] connections, available `options` are:
   See [Identifying paths for IPC connections][]. If provided, the TCP-specific
   options above are ignored.
 
+For both types, available `options` include:
+
+* `onread` {Object} - If specified, incoming data is stored in a single `buffer`
```

Comment:
An example showing this in use would be ideal.

Reply:
Added. Let me know if it suffices.

Reply:
Thank you! looks good